libs/@local/hash-subgraph/jest.config.cjs

```diff
@@ -13,7 +13,7 @@ module.exports = {
   // The chosen default give a lot of room to the integration test.
   testTimeout: 60000,
   moduleNameMapper: {
-    "@hashintel/hash-graph-client": "<rootDir>/../../graph/clients/typescript",
+    "@local/hash-graph-client": "<rootDir>/../../graph/clients/typescript",
```

Comment:
I believe we should adjust this
```suggestion
    "@local/hash-graph-client": "<rootDir>/../hash-graph-client",
```
Although I don't think it affects anything as we're just importing types which are erased during the tests